sips/sip-022/sip-022-emergency-pox-fix.md

```diff
@@ -0,0 +1,284 @@
+            add-amount: (get add-amount data) })))))
+```
+
+The bug is reachable by any solo Stacker who calls `stack-increase`.  Any solo
+Stacker who increases their total STX locked will erroneously set the amount of
+uSTX backing their PoX address to be equal to the current total number of uSTX
+locked in this cycle (the value `total-ustx`), instead of the sum of their
+current locked uSTX amount and their added amount.  The act of triggering this
+bug is an unavoidable consequence of calling the `stack-increase` function --
+individuals who call `stack-increase` are not thought to be deliberately
+exploiting the system.
+
+This bug was first triggered in the wild by Stacks transaction
+`20e708e250bad3fb5d5ab84a70365c3c1cf0520c7a9f67cd5ff6b9fa94335ea5`.
+
+The immediate consequences for PoX of this bug being triggered are as follows:
+
+* The total STX locked to the PoX address owned by the caller of `stack-increase`
+  will be potentially higher than the amount of STX that the caller possesses.
+The caller will receive PoX payouts _as if_ they had locked that much STX.  So,
+the caller receives undue BTC.
+
+* The stacking threshold is raised to account for the PoX contract's reported
+  increase in STX locked.
+
+Similar consequences are expected when a stacker contributes to two different PoX addresses and 
+at least one PoX address would benefit from auto-unlocking. This behaviour has not been seen in the wild.
+
+
+Furthermore, if this bug is triggered enough times, the Stacks network will crash.  This is
+because of a runtime assertion in the PoX reward set calculation logic that
+verifies that the total locked STX does not exceed the total liquid STX.  This
+would no longer be true due to this bug.  The offending assertion is detailed
+below:
+
+```rust
+pub fn get_reward_threshold_and_participation(
+    pox_settings: &PoxConstants,
+    addresses: &[RawRewardSetEntry],
+    liquid_ustx: u128,
+) -> (u128, u128) {
+    let participation = addresses
+        .iter()
+        .fold(0, |agg, entry| agg + entry.amount_stacked);
+
+    assert!(
+        participation <= liquid_ustx,
+        "CORRUPTION: More stacking participation than liquid STX"
+    );
+```
+
+The `RawRewardSetEntry` data are pulled directly from the
+`reward-cycle-pox-address-list` data map, and the `.amount_stacked` field is
+equal to the `total-ustx` field that was erroneously set in `stack-increase`.
+
+Considering these consequences with respect to the [draft blockchain catastrophic
+failure and recovery guidelines](https://github.com/stacksgov/sips/pull/10),
+this bug would be categorized as requiring an _immediate_ hard fork to rectify.
+The network has not yet crashed, but it is in imminent danger of crashing and
+there is insufficient time to execute a Stacker-based SIP vote as has been customary for
+past hard forks.  This SIP instead proposes that this hard fork activate at the
+start of the next reward cycle (#58) at the time of this writing.
+
```

Comment:
In lieu of a community vote, perhaps it would be appropriate to summon the Stack Foundation board as a stand-in.

Reply:
that's an interesting idea - would the various CAB's voting on this SIP also suffice here vs a board vote?

Reply:
* Stacks Foundation boards is only for the Foundation entity, not boards for the rest of ecosystem is how I read it.
* If we want something like that would be more appropriate to have ecosystem "decentralized boards", which is kind of what Steering Committee is acting as too? Would cross-over responsibility if that's the idea. Option is to have more members on this SC committee. 
* Currently: Requires A. SIP Editor approval to "Accept", CAB votes approval to "Recommend", Steering Committee approval to become "Activation-in-Progress" 
as per SIP-000: https://github.com/stacksgov/sips/blob/main/sips/sip-000/sip-000-stacks-improvement-proposal-process.md